subjects/java/raids/Jaikin/README.md

```diff
@@ -0,0 +1,37 @@
+## Implementation of Jaikin's Algorithm with Step-by-Step Animation
```

Comment:
I would rename this as `Jaikin` or change the sentence by saying 
`Implementation of Chaikin's .....` 

based on what you shared [here](https://github.com/01-edu/public/pull/2177#discussion_r1513044407)